### DISCUSSION TITLE
Debug mode is not working anymore

### DISCUSSION THREAD

**Discussion by vinlos** [lucono/karma-test-explorer#104] [Issues / Help]
I am not able to run tests in debug mode. I've tried with different repositories, settings, machines and operating systems, but, every time, tests got stuck and executions never reach a breakpoint
```
[2024-11-05 23:26:45.832] [INFO] [KarmaTestExplorer]: Starting debug session
[2024-11-05 23:26:46.121] [INFO] [Debugger]: Debug session 'Karma Test Explorer Debugging - frontend' has started
[2024-11-05 23:26:46.122] [INFO] [KarmaTestExplorer]: Debug test run started
```

**Comment by malcolms** [comment]
This may be an issue with vscode 1.95 as several similar issues have been raised against the vscode project:

https://github.com/microsoft/vscode/issues/233524
https://github.com/microsoft/vscode/issues/233716
https://github.com/microsoft/vscode/issues/233748

**Comment by server-mm404** [comment]
> I am not able to run tests in debug mode. I've tried with different repositories, settings, machines and operating systems, but, every time, tests got stuck and executions never reach a breakpoint
> 
> ```
> [2024-11-05 23:26:45.832] [INFO] [KarmaTestExplorer]: Starting debug session
> [2024-11-05 23:26:46.121] [INFO] [Debugger]: Debug session 'Karma Test Explorer Debugging - frontend' has started
> [2024-11-05 23:26:46.122] [INFO] [KarmaTestExplorer]: Debug test run started
> ```

**Comment by vinlos** [comment]
After some time, this is working again. I don't know why. Probably this was fixed with one of the latest VsCode versions.
